packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2189,15 +2189,18 @@ export default class Meeting extends StatelessWebexPlugin {
    */
   setLocalAudioTrack(audioTrack, emitEvent = true) {
     if (audioTrack) {
+      // TODO - Add logic for more settings
       const settings = audioTrack.getSettings();
 
+
       this.mediaProperties.setMediaSettings('audio', {
         echoCancellation: settings.echoCancellation,
         noiseSuppression: settings.noiseSuppression
       });
 
       LoggerProxy.logger.log('Meeting:index#setLocalAudioTrack --> Audio settings.', JSON.stringify(this.mediaProperties.mediaSettings.audio));
       this.mediaProperties.setLocalAudioTrack(audioTrack);
+      this.audioTrack = audioTrack;
```

Comment:
The values are already present in this.mediaproperties.audioTrack we dont need to save it again